jdat_notebooks/ifu_optimal/requirements.txt

```diff
@@ -0,0 +1,12 @@
+notebook
+jupyter_client==5.3.5
```

Comment:
I would add specific versions of each of these. Moving forward there might come incompatablities and we would have a hard time figuring the current versions out.

Reply:
@robelgeda 
Should the glue, glue-jupyter, and jdaviz versions also be explicit?  Currently they point to the latest:
git+https://github.com/glue-viz/glue.git
git+https://github.com/glue-viz/glue-jupyter.git
git+https://github.com/spacetelescope/jdaviz

Reply:
I think this is okay as is. It's because we want to make sure the notebook works with the latest version of the viz tools when the CI builds this notebook.

Reply:
Added specific versions for most requirements.